packages/maskbook/src/plugins/Trader/apis/coinmarketcap/index.ts

```diff
@@ -272,3 +273,10 @@ export async function getLatestMarketPairs(id: string, currency: string) {
     }
 }
 //#endregion
+
+//#region get price
+export function getPrice(coinId: number) {
```

Comment:
export { getCoinPrice as getPrice } from '...'

Reply:
this is debug code